Projects/Contracts/Sources/View/ContractRow.swift

```diff
@@ -105,38 +87,23 @@ struct ContractRowButtonStyle: SwiftUI.ButtonStyle {
             }
             Spacer()
             HStack {
-                hText(contract.displayName, style: .title2)
+                hText(contract.displayName)
                 Spacer()
             }
-            HStack {
-                ForEach(contract.detailPills, id: \.self) { pill in
-                    if contract.gradientOption == nil {
-                        DetailPill(text: pill)
-                            .overlay(
-                                RoundedRectangle(cornerRadius: 4)
-                                    .stroke(hLabelColor.primary, lineWidth: 1)
-                            )
-                    } else {
-                        DetailPill(text: pill)
-                    }
-                }
-                Spacer()
-                ContractRowChevron()
-            }
+            hText(contract.getDetails())
+                .foregroundColor(hTextColorNew.negative.opacity(0.415))
```

Comment:
Should be hGrayscaleTranslucentDarkColorNew.greyScaleTranslucent700

Reply:
Updated color :D